src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/csip/Transformer.kt

```diff
@@ -78,6 +78,19 @@ private fun CSIPReport.toSCSResponse() =
     reasonForDecision = reasonForDecision,
   )
 
+fun CSIPFactor.toFactorResponse() =
+  CSIPFactorResponse(
+    id = id,
+    type = type.toCodeDescription(),
+    comment = comment,
+    createDateTime = createDatetime,
+    createdBy = createUsername,
+    createdByDisplayName = createdByStaffUserAccount.asDisplayName(),
```

Comment:
so is createdByStaffUserAccount guaranteed *never* to be null ? is there a foreign key constraint or whatever?

Reply:
yes - it works for null.